server/odc-service/src/main/java/com/oceanbase/odc/service/session/ConnectConsoleService.java

```diff
@@ -238,7 +238,9 @@ public SqlAsyncExecuteResp streamExecute(@NotNull String sessionId, @NotNull @Va
     public SqlAsyncExecuteResp streamExecute(@NotNull String sessionId,
             @NotNull @Valid SqlAsyncExecuteReq request, boolean needSqlRuleCheck) throws Exception {
         ConnectionSession connectionSession = sessionService.nullSafeGet(sessionId, true);
-
+        if (ConnectionSessionUtil.getLogicalSession(connectionSession)) {
+            return new SqlAsyncExecuteResp(true);
```

Comment:
Okay, I will implement a new SqlInterceptor. However, it's not appropriate to throw an exception here, because this scenario itself is not an error/exception, similar to other SQL interception scenarios.

Reply:
Oh I recalled why I didn't implement a new `SqlInterceptor`. Before SqlInterceptor.preHandle is executed, many other operations are performed, such as splitting SQLs and constructing kill session SQLs. These operations are useless for the logical database and might even cause errors. Therefore, I prefer to maintain the current method

Reply:
ok